tools/failures.py

```diff
@@ -62,8 +70,9 @@ def format_in_table(active_jobs, master):
     sum_removed = 0
     sum_remaining = 0
 
-    data = requests.get('http://alertmanager.allizom.org/data/jobnames/',
-                        headers={'accept-encoding': 'json'}, verify=True).json()
+    data = retry(requests.get, args=('http://alertmanager.allizom.org/data/jobnames/', ),
```

Comment:
hmm, I don't think so. Because when SETA visit this url, it actually visiting this url(https://github.com/mozilla/ouija/pull/93/files#diff-94ab7c76892f0032ee9ad040e3e8f207R496). And it will return a list of jobnames stored in jobtypes.py :)